homework3/semi-uber/UI/src/environments/environment.ts

```diff
@@ -4,7 +4,10 @@
 
 export const environment = {
   production: false,
-  regUrl: "http://localhost:3000/registration"
+  regUrl: "http://localhost:3000/registration",
+  pasRegexp:
```

Comment:
Bad idea. regular expressions are not dependent on environment, so there is no reason to put them to env variables

Reply:
fixed